source/manage/mmctl-command-line-tool.rst

```diff
@@ -2996,7 +2996,12 @@ mmctl ldap idmigrate
 
 **Description**
 
-Migrate LDAP ``IdAttribute`` to a new value. Run this utility then change the IdAttribute to the new value.
+Migrate LDAP ``IdAttribute`` to a new value. Run this utility to change the value of your ID Attribute without your users losing their accounts. After running the command you can change the ID Attribute to the new value in the System Console. For example, if your current ID Attribute was ``sAMAccountName`` and you wanted to change it to ``objectGUID``, you would:
+
+- Wait for an off-peak time when your users won’t be impacted by a server restart.
```

Comment:
If this has to be performed in this sequence, I'm wondering if a numbered list may work better?

Reply:
Up to you :)

Yes it is a sequence

Reply:
should be ok